mappings/net/minecraft/world/gen/feature/TwistingVinesFeature.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4953 net/minecraft/world/gen/feature/TwistingVinesFeature
+	METHOD method_25986 isNotSuitable (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_25987 generateColumn (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338$class_2339;III)V
+		ARG 0 world
+		ARG 1 random
+		ARG 2 pos
+		ARG 3 maxLength
+		ARG 4 minAge
+		ARG 5 maxAge
+	METHOD method_25988 generatePile (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338;)V
```

Comment:
```suggestion
	METHOD method_25988 tryGenerateVines (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338;)V
```

Reply:
No, it generates multiple columns of vines in this method.